README.md

```diff
@@ -82,14 +146,9 @@ to use, test, and provide feedback on the new features are encouraged to try
 this branch. However, this branch provides no stability guarantees and reserves
 the right to change and break the API at any time.
 
-## Further knowledge
-
-It may help to read the [Google Test documentation](https://github.com/google/googletest/blob/master/googletest/docs/primer.md)
-as some of the structural aspects of the APIs are similar.
-
-## Example usage
+## Usage
 ### Basic usage
-Define a function that executes the code to be measured, register it as a
+Define a function that executes the code to benchmark, register it as a
```

Comment:
I think i'm missing some subtle difference here?
It does measure the time it takes to execute the code, computes some specified metrics.

Reply:
Style edit. Perhaps not an improvement?

Reply:
I'm fine either way on this.